services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/impl/DeviceRegistryConfigProperties.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+
+package org.eclipse.hono.deviceregistry.impl;
+
+import java.util.Objects;
+
+import org.eclipse.hono.config.ServiceConfigProperties;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+
+/**
+ * Configuration properties for the device registry as own server.
+ *
+ */
+public class DeviceRegistryConfigProperties extends ServiceConfigProperties {
```

Comment:
this should probably be final, shouldn't it. At least this class is not supposed to be extended by client code AFAIC